category_theory/limits/limits.lean

```diff
@@ -373,6 +373,9 @@ begin
   refl
 end
 
+def lim_yoneda : lim ⋙ yoneda ≅ category_theory.cones J C :=
+nat_iso.of_components (λ F, nat_iso.of_components (limit.hom_iso F) (by tidy)) (by tidy)
+
```

Comment:
That's surprisingly pleasant to prove!

Reply:
Yes, I was also very happy. It's sad that the dual is slightly more annoying.